### DISCUSSION TITLE
I get this error when uploading

### DISCUSSION THREAD

**Discussion by Richskinz** [naverz/zepeto-studio-kor#501] [Issue Report (Legacy)]
I keep getting this error when I upload my item 

![Screenshot_20220316-021635_YouTube](https://user-images.githubusercontent.com/101695911/158531401-650b8e0d-6227-4743-9705-639d92ddd09d.png)

**Comment by Zepeto-TechEvangelist** [comment]
As the error message says, it seems to be an item modeling error.
Did you try following the "View guide"?

**Comment by Richskinz** [comment]
Please are u going to help me?

**Reply by Zepeto-TechEvangelist** [reply]
Please show me the appearance of the item and the name of the item.
I'm not a person who checks items, so I don't know exactly what the problem is just by reading the text.

**Comment by Richskinz** [comment]
I need someone to see the item please o beg u to help me

**Reply by Zepeto-TechEvangelist** [reply]
Please screen capture the item you worked on and upload it here.
Also, please let me know what category it is.

**Comment by Richskinz** [comment]
The item on zepeto

https://studio.zepeto.me/preview/6230a30b97c4a8545f0052ac

**Reply by Zepeto-TechEvangelist** [reply]
The error is due to the number of joints exceeding 100.
Please reconfirm the number of joints.
You can get a lot of information by googling about it.

This is a community that posts information about ZEPETO world development.
In the future, please contact the customer service center for this issue.
https://support.zepeto.me/hc/en-us/requests/new

**Comment by Richskinz** [comment]
Yes , I did everything that was told to me. I told that it was because the
helmet chnage with hair color , I read that link u sent me and still after
doing all you asked my item got rejected.
I have been promoting zepeto for a long time and I'm telling you I think
you all have a bug in your system 😔 or someone isnt looking good , my item
is 100% on all rules I sent item file to u, please I beg u to help me.
